common/djangoapps/track/middleware.py

```diff
@@ -36,6 +36,22 @@
 }
 
 
+class ResponseLoggingMiddleware(object):
+    """
+    A debugging middleware for the purpose of understanding what the current state of the response
+    is at this point in the stack.
+    """
+
+    def process_response(self, response):
+        """
+        Logs the response at the current point in the middleware stack for debugging purposes.
+        """
+        log.info('Logging response for debugging purposes: %r', response)
```

Comment:
Can we put the whole thing in a try, except and swallow any issues in this code, in case we get something wrong we still want the rest of the code to work as is.